rustls-platform-verifier/src/verification/apple.rs

```diff
@@ -72,21 +72,30 @@ impl Verifier {
     /// facilities with the addition of extra root certificates to trust.
     ///
     /// See [Verifier::new] for the external requirements the verifier needs.
-    pub fn new_with_extra_roots(roots: Vec<pki_types::CertificateDer<'static>>) -> Self {
-        Self {
-            extra_roots: roots,
+    pub fn new_with_extra_roots(
+        roots: Vec<pki_types::CertificateDer<'static>>,
+    ) -> Result<Self, TlsError> {
```

Comment:
how about bumping the crate version to `0.5.0` in this commit, so we don't forget the next release is a breaking one?

Reply:
We already bump the version to 0.4 for this reason https://github.com/rustls/rustls-platform-verifier/pull/133/commits/094341ed7c8de588fdeceabdadd397be9cc371c2

Reply:
Ahh! Good catch, sorry about missing that. :+1:

Reply:
no problem ;)

Reply:
Ah, great!